signup/forms.py

```diff
@@ -197,10 +208,22 @@ class PublicKeyForm(forms.Form):
         strip=False,
         help_text=password_validation.password_validators_help_text_html())
 
+    user_password = forms.CharField(
```

Comment:
Why `user_password` here and not `password` as done in the other end points?

Reply:
Because `password` field was already used here. I was thinking to replace `password` with `passphrase` initially but wasn't sure whether this field was related to a public key or to LDAP.